staging/src/k8s.io/api/storage/v1beta1/types.go

```diff
@@ -364,11 +364,7 @@ type CSIDriverSpec struct {
 	//
 	// This field was immutable in Kubernetes <= 1.22 and now is mutable.
 	//
-	// This is a beta field and only available when the CSIStorageCapacity
```

Comment:
Good catch, I missed that one.

While editing I also noticed that I should have used CSIDriverSpec.StorageCapacity instead of CSIDriver.StorageCapacity - also fixed.